microsoft-edge/playwright/index.md

```diff
@@ -76,7 +76,7 @@ const config: PlaywrightTestConfig = {
     {
       name: 'Microsoft Edge',
       use: {
-        // Supported Microsoft Edge channels are: msedge, msedge-beta, msedge-dev, msedge-canary
+        // Supported Microsoft Edge channels are: msedge, msedge-beta, msedge-dev
```

Comment:
Is there a URL for the Playwright docs where it states that only Stable, Dev, and Beta are supported, not Canary?

Reply:
@kumardeepakr3 @bwalderman @zoherghadyali - Is there a URL for the Playwright docs where it states that only Stable, Dev, and Beta are supported, not Canary?

Reply:
I can't find where this is stated in the https://playwright.dev docs. The API docs show that `"msedge-canary"` is supported. E.g.: https://playwright.dev/docs/api/class-testoptions#test-options-channel

Reply:
The step that directly follows the code snippet fails for 'msedge-canary'. Running 'npx playwright install msedge-canary' throws error. However when someone installs canary directly and uses it in the code snippet it works. My bad, abandoning the PR.

![image](https://user-images.githubusercontent.com/8517001/150785037-c63c0805-413b-4e12-af53-9f80585008db.png)